### DISCUSSION TITLE
Lots of log output when initializing a gradle project

### DISCUSSION THREAD

**Discussion by ahus1** [JetBrains/Qodana#115] [Q&A] [answered]
When running my GitHub action on a project with the cache not initialized yet, I see a lot of messages, probably when downloading the dependencies. 

```
2022-08-13T14:53:55.8789523Z (Project configuration) Download https://cache-redirector.jetbrains.com/www.jet
```

This amounts to several 100,000 log lines. The log lines also seem to be truncated. 

Is this is for me difficult to analyze (due to the truncation and of the line and the lots of output) and might slow down the initialization. 

Is this something that I could change, or is it something that Qodana can improve? Thanks!

The workflow: https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/main/.github/workflows/docs.yml

The log (35 MB uncompressed): [2.zip](https://github.com/JetBrains/Qodana/files/9332019/2.zip)

Thanks!

**Comment by tiulpin** [comment, marked as answer]
Hello, Alexander!
Sorry for the confusion, we are planning to improve this output in the next release.

The best available way to check the logs at the moment would be inspecting `idea.log` (in the results artifact)

**Reply by ahus1** [reply]
Thanks for pointing me at the artifact. I found the `gradle-import.log`: 

```
Download https://cache-redirector.jetbrains.com/www.jetbrains.com/intellij-repository/snapshots/com/jetbrains/intellij/idea/ideaIC/222.3739.24-EAP-SNAPSHOT/ideaIC-222.3739.24-EAP-SNAPSHOT.zip (15.75 kB / 679.01 MB)
Download https://cache-redirector.jetbrains.com/www.jetbrains.com/intellij-repository/snapshots/com/jetbrains/intellij/idea/ideaIC/222.3739.24-EAP-SNAPSHOT/ideaIC-222.3739.24-EAP-SNAPSHOT.zip (17.9 kB / 679.01 MB)
```

So it prints a new line for every few kilobytes it downloaded.  

Good to hear you're on this!
